Doc/tools/touch-clean-files.py

```diff
@@ -0,0 +1,17 @@
+#!/usr/bin/env python3
+"""
+Touch files that must pass Sphinx nit-picky mode
+so they are rebuilt and we can catch regressions.
+"""
+
+from pathlib import Path
+
+CLEAN = [
+    "Doc/library/gc.rst",
+    "Doc/library/sqlite3.rst",
+    "Doc/whatsnew/3.12.rst",
+]
+print("Touching:")
+for filename in CLEAN:
+    print(f"{filename}")
```

Comment:
Since these are already strings, why not

```suggestion
    print(filename)
```

Reply:
Of course :) I started off with "Touching " in there too but forgot to de-eff it.

Reply:
Hehe, figured it was something like that